Sources_v3/StreamChat/Config/Token.swift

```diff
@@ -1,40 +1,41 @@
 //
-// Copyright © 2020 Stream.io Inc. All rights reserved.
+// Copyright © 2021 Stream.io Inc. All rights reserved.
 //
 
 import Foundation
 
-/// A token is used to authenticate a user.
-public typealias Token = String
+enum TokenError: Error {
+    case missingUserId
+}
```

Comment:
nit: Please use the `ClientError` approach we use for all other errors.

Reply:
done